nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -545,17 +542,14 @@ impl<'a> BlueprintBuilder<'a> {
                 .blueprint_disks
                 .get(sled_id)
                 .cloned()
-                .unwrap_or_else(|| BlueprintPhysicalDisksConfig {
-                    generation: Generation::new(),
-                    disks: Vec::new(),
-                });
+                .unwrap_or_else(|| BlueprintPhysicalDisksConfig::default());
```

Comment:
Is it weird that `BlueprintPhysicalDisksConfig` impls `Default` but `BlueprintDatasetsConfig` doesn't? (Does `BlueprintZonesConfig`?)

I'm not sure whether I'd vote to add default to the ones missing it or remove it from the one(s?) that have it...

Reply:
I agree it's kinda weird, but I just went with what existed. I'm not sure which one to vote for either. I'd usually vote for getting rid of `Default`, but it sure seems handy here. 

To avoid another CI lap I merged this in, but I'm happy to fix this in the follow up code I'm working on.